frontend/src/i18n/lang/ar-SA.json

```diff
@@ -600,7 +600,8 @@
 				"color": "تحديد اللون",
 				"delete": "حذف",
 				"favorite": "أضف إلى التفضيلات",
-				"unfavorite": "حذف من التفضيلات"
+				"unfavorite": "حذف من التفضيلات",
```

Comment:
Please do not add translations here, they will be synced from Crowdin. Only add the new source string in `en.json`.

Check out the docs: https://vikunja.io/docs/translations/